src/templates/company/index.js

```diff
@@ -1,12 +1,13 @@
 /* eslint-disable react/prop-types */
 import React from 'react'
+import BN from 'bn.js'
 import { ClaimDomain, Review, Voting, Tokens } from '../kit'
 
 import header from './header.svg'
 import icon from './icon.svg'
 
 export default {
-  id: 'company',
+  id: 'company-template.aragonpm.eth',
```

Comment:
@bpierre I've repurposed each template's `id` to their aragonPM repo domains, since we need it anyway for ENS resolution.